ExtractorUtils/BaseExtractor.cs

```diff
@@ -293,5 +304,17 @@ public void Dispose()
             Dispose(disposing: true);
             GC.SuppressFinalize(this);
         }
+
+        /// <summary>
+        /// Dispose extractor asynchronously. Preferred.
```

Comment:
Preferred over what alternative, and is that referencing this function as the better choice?